propertyestimator/workflow/protocols.py

```diff
@@ -377,6 +385,44 @@ def merge(self, other):
 
         return {}
 
+    def get_value_references(self, input_path):
+        """Returns a list of references to the protocols which one of this
+        protocols inputs (specified by `input_path`) takes its value from.
+
+        Notes
+        -----
+        Currently this method only functions correctly for an input value which
+        is either currently a :obj:`ProtocolPath`, or a `list` which contains a
+        :obj:`ProtocolPath`.
+
+         Parameters
+         ----------
+         input_path: :obj:`ProtocolPath`
+            The input value to check.
+
+        Returns
+        -------
+        List[:obj:`ProtocolPath`]
```

Comment:
Oh cool. I bet this plays well with Sphinx -- Will need to check this out.

Reply:
You'd think so... but at the moment doesn't!